kmip/pie/client.py

```diff
@@ -541,6 +542,59 @@ def activate(self, uid=None):
             message = result.result_message.value
             raise exceptions.KmipOperationFailure(status, reason, message)
 
+    def revoke(self, code, uid=None, message=None,
```

Comment:
Ok, scratch what I said above. Let's rename `code` to `revocation_reason` and let's rename `message` to `revocation_message`. That's clearer and preserves the structure of the `RevocationReason` defined in the spec.